### PR TITLE
Deleted ASP.NET from the list

### LINKED ISSUE #37244: ASP.NET is not a server-side web language, it is a web framework that supports the programming languages C#, F# and VB!
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/What_is_JavaScript

### What specific section or headline is this issue about?

Server-side versus client-side code

### What information was incorrect, unhelpful, or incomplete?

In the documents it is written that:

> Examples of popular server-side web languages include PHP, Python, Ruby, ASP.NET, and even JavaScript!

ASP.NET is NOT a language. It is a web framework which is used to build web apps and services using .NET and C#.
If I make an analogy, ASP.NET is like Express.js framework, .NET SDK is like Node.js runtime, and C# is the programming language like JavaScript.

### What did you expect to see?

You may write C# instead of ASP.NET.

### Do you have any supporting links, references, or citations?

https://dotnet.microsoft.com/en-us/apps/aspnet
https://dotnet.microsoft.com/en-us/learn/dotnet/what-is-dotnet
https://dotnet.microsoft.com/en-us/languages/csharp

### Do you have anything more you want to share?

ASP NET Core

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/javascript/first_steps/what_is_javascript`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/What_is_JavaScript
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/javascript/first_steps/what_is_javascript/index.md
* Last commit: https://github.com/mdn/content/commit/245715b48674c1729cb63417e4a27628e30ae28c
* Document last modified: 2024-08-03T13:26:37.000Z

</details>

### PR DESCRIPTION
**Pull request by asadullahehsan**
According to issue https://github.com/mdn/content/issues/37244, I am going to delete ASP.NET from the list of server-side web programming languages.

### Description

I am deleting the ASP.NET from the list of web programming languages, as discussed in issue #37244.

### Motivation

Please refer to issue #37244 

### Additional details

https://github.com/mdn/content/issues/37244#issuecomment-2549778884

### Related issues and pull requests

Fixes #37244

### CODE REVIEW

File: files/en-us/learn/javascript/first_steps/what_is_javascript/index.md [hunk 1]
```diff
@@ -192,7 +192,7 @@ Client-side code is code that is run on the user's computer — when a web page
 In this module we are explicitly talking about **client-side JavaScript**.
 
 Server-side code on the other hand is run on the server, then its results are downloaded and displayed in the browser.
-Examples of popular server-side web languages include PHP, Python, Ruby, ASP.NET, and even JavaScript!
+Examples of popular server-side web languages include PHP, Python, Ruby, and even JavaScript!
```
Comment (Josh-Cena) [inline files/en-us/learn/javascript/first_steps/what_is_javascript/index.md:195]: All these languages are interpreted and ill-performing. Suggest to add at least one compiled language, let it be Go, C#, or Java; @wbamberg

Comment (wbamberg) [reply files/en-us/learn/javascript/first_steps/what_is_javascript/index.md:195]: I'm happy with that.

Comment (Josh-Cena) [reply files/en-us/learn/javascript/first_steps/what_is_javascript/index.md:195]:
```suggestion
Examples of popular server-side web languages include PHP, Python, Ruby, C#, and even JavaScript!
```

**Review by wbamberg — APPROVED** [review]
👍 thank you, @asadullahehsan and @Josh-Cena !

**Pull request merged**